docs/GCP-development.md

```diff
@@ -257,10 +278,10 @@ tkn pipelineruns logs build-pipeline-run -f
 If you have tekton dashboard installed
 
 ```bash
-kubectl port-forward svc/tekton-dashboard 9097:9097 -n tekton-pipelines
+kubectl port-forward svc/tekton-dashboard 9097:9097 -n tekton-pipelines &
```

Comment:
For consistency, maybe move this to a location like in the local doc?

Reply:
I dont think kind setup installs the dashboard?

Reply:
Yeah it does, and I validated it works :) 
https://github.com/Cray-HPE/sigstore-poc/blob/main/hack/kind/setup-kind.sh#L400